docs/i18n.md

```diff
@@ -1,5 +1,121 @@
 # Internationalization
 
-To make text translatable, you’d use the `translate()` function from [i18n-calypso](../packages/i18n-calypso/README.md). We extract the translatable texts wrapped inside the function and write a POT file that is imported to [translate.wordpress.com](https://translate.wordpress.com/).
+If you add or modify text strings that will be displayed to users, consider whether they should be visible to non-English users.
+In that case, wrap the strings in the `translate()` function from [i18n-calypso](../packages/i18n-calypso/README.md).
+We extract the translatable texts wrapped inside the function and write a POT file that is exported to [translate.wordpress.com](https://translate.wordpress.com/).
```

Comment:
Maybe this should link directly to the [WordPress.com](https://translate.wordpress.com/projects/wpcom/) Project?